recipes/affogato/build.sh

```diff
@@ -0,0 +1,41 @@
+##
+## START THE BUILD
+##
+
+mkdir -p build
+cd build
+
+# Set the correct python flags, depending on the distribution
+PY_VER=$(python -c "import sys; print('{}.{}'.format(*sys.version_info[:2]))")
+PY_ABIFLAGS=$(python -c "import sys; print('' if sys.version_info.major == 2 else sys.abiflags)")
+PY_ABI=${PY_VER}${PY_ABIFLAGS}
+
+##
+## Configure
+##
+cmake .. \
+        -DCMAKE_C_COMPILER=${CC} \
+        -DCMAKE_CXX_COMPILER=${CXX} \
+        -DCMAKE_BUILD_TYPE=RELEASE \
+        -DCMAKE_OSX_DEPLOYMENT_TARGET=${CMAKE_OSX_DEPLOYMENT_TARGET}\
+        -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+        -DCMAKE_PREFIX_PATH=${PREFIX} \
+\
+        -DCMAKE_SHARED_LINKER_FLAGS="${LDFLAGS}" \
+        -DCMAKE_EXE_LINKER_FLAGS="${LDFLAGS}" \
+        -DCMAKE_CXX_FLAGS="${CXXFLAGS}" \
+        -DCMAKE_CXX_FLAGS_RELEASE="${CXXFLAGS} -O3 -DNDEBUG" \
+        -DCMAKE_CXX_FLAGS_DEBUG="${CXXFLAGS}" \
+\
+        -DBOOST_ROOT=${PREFIX} \
+        -DBUILD_PYTHON=ON \
+\
```

Comment:
I would have expected that the following should suffice:
```suggestion
    ${CMAKE_ARGS} \
```